.github/workflows/call-build-macos.yaml

```diff
@@ -75,7 +75,10 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: ${{ inputs.ref }}
-
+      - name: Install dependencies
+        run: |
+          brew update
+          brew install bison flex || true
```

Comment:
Why do we need the true check? Does it fail if already present?

Reply:
Yes. We got failure if the bison or flex is already installed.

Reply:
Great, MacOS different to all other things! 
Can you just add a comment to clarify then?